content/docs/rfcs/11/README.md

```diff
@@ -9,45 +9,63 @@ contributors:
   - Sanaz Taheri <sanaz@status.im>
 ---
 
-`WakuRelay` is part of the gossip domain for Waku. It is a thin layer on top of GossipSub.
+<!-- A note to contributors:
+Please make use of the following best practices when adding to this document:
+- semantic line breaks as per https://sembr.org/
+- RFC key words as per RFC 2119 (https://www.ietf.org/rfc/rfc2119.txt) -->
 
-**Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
+`WakuRelay` specifies a [Publish/Subscribe approach](https://docs.libp2p.io/concepts/publish-subscribe/) to peer-to-peer messaging with a strong focus on privacy, censorship-resistance, security and scalability.
```

Comment:
Instead if WakuRelay I'd call this 11/WAKU2-RELAY (ditto other parts of the spec)

Reply:
Done in aeae2b5